app/controllers/concerns/exception_handler.rb

```diff
@@ -6,5 +6,24 @@ module ExceptionHandler
       Rails.logger.error("Not found: #{err.message}")
       json_response({:message => "Not Found"}, :not_found)
     end
+
+    rescue_from Catalog::TopologyError do |err|
+      json_response({:message => err.message}, :internal_server_error)
```

Comment:
Should we log the `err.message` here? I know we weren't before - but it probably wouldn't hurt since we're doing it for the other errors.

Reply:
@lindgrenj6 I *think* the only place this error comes from is `lib/topological_inventory.rb`, and it's already logging the message here https://github.com/ManageIQ/catalog-api/blob/master/lib/topological_inventory.rb#L15

Reply:
Oh yep you're right. I should have looked into that more. Coolio!